epidose/device/util.sh

```diff
@@ -186,6 +196,24 @@ get_new_filter()
   fi
 }
 
+# Check and update software if necessary
+# preconditions: WiFi should be turned on
+check_for_updates()
+{
+  log "Checking for updates"
+  remote_updates_available=false
+ 
+  # Set username and email because git asks for it
+  git config --system user.name "$USERNAME"
+  git config --system user.email "$EMAIL"
```

Comment:
These should be done once, during setup.

Reply:
Ok, I will add those lines in the Ansible file then.